app/src/main/java/org/wikipedia/feed/FeedContentType.java

```diff
@@ -88,7 +88,7 @@ public FeedClient newClient(AggregatedFeedContentClient aggregatedClient, int ag
             return isEnabled() ? new BecauseYouReadClient() : null;
         }
     },
-    SUGGESTED_EDITS(9, R.string.suggested_edits_feed_card_title, R.string.feed_item_type_suggested_edits, false) {
+    SUGGESTED_EDITS(9, R.string.suggested_edits_feed_card_title, R.string.feed_item_type_suggested_edits, false, AccountUtil.isLoggedIn()) {
```

Comment:
Hang on, sorry, @sharvaniharan's previous comment is not correct for this case.  This doesn't actually work because it will make the state of this item permanent. (If you log in afterwards, the item will not appear, because the constructor was already called.)  Your original method was just fine.

Reply:
Done!